fancontrol/fancontrol.cpp

```diff
@@ -82,20 +84,18 @@ FANCONTROL::FANCONTROL(HINSTANCE hinstapp)
 	Runs_as_service(FALSE),
 	ActiveMode(false),
 	ManFanSpeed(7),
-	UseTWR(0),
```

Comment:
do not remove TWR (used to see more sensors for debugging)

Reply:
I see. These are actual mirrors of the 0.6.2 ports in the 0.6.3 paradigm. I would, if keeping that, want to recode that to the same way all the other ports are being read.

Or not. Not my decision.

Reply:
It is kludge code that should be moved to portio.cpp as a separate cleaned up function.

**Reinstatet asis**

Reply:
Yea that's a good point, should prob be put on the to-do list